src/main/flight/pid.h

```diff
@@ -118,6 +118,7 @@ typedef struct pidProfile_s {
     uint8_t acro_trainer_angle_limit;       // Acro trainer roll/pitch angle limit in degrees
     uint16_t acro_trainer_lookahead_ms;     // The lookahead window in milliseconds used to reduce overshoot
     uint8_t acro_trainer_debug_axis;        // The axis for which record debugging values are captured 0=roll, 1=pitch
+    uint8_t acro_trainer_gain;              // The strength of the limiting. Raising may reduce overshoot but also lead to oscillation around the angle limit
```

Comment:
Maybe use LEVEL mode gain?

Reply:
At the moment the default gain I'm using is different then level mode gain.  In the long term I'm hoping to eliminate the gain entirely for a well chosen hardcoded value.  Since the target of this feature is the beginning pilot I think have as few configurables as possible is desirable.  If we can't find a reasonable gain that works well in most cases then having the value shared (and hidden) under level mode would be confusing.

Reply:
The idea was that reasonably-tuned level mode will provide good acro trainer behavior ...

But with angle-rate PID, level gain may be quite insensitive to quad parameters